test_projects/django/testproj/urls.py

```diff
@@ -1,8 +1,10 @@
 from django.conf.urls import patterns, include, url
 
-from djcelery.views import apply
-
-urlpatterns = patterns('',
-    url(r'^apply/(?P<task_name>.+?)/', apply, name='celery-apply'),
-    url(r'^celery/', include('djcelery.urls')),
-)
+try:
+    from djcelery.views import apply
+    urlpatterns = patterns('',
+        url(r'^apply/(?P<task_name>.+?)/', apply, name='celery-apply'),
+        url(r'^celery/', include('djcelery.urls')),
+    )
+except ImportError:
+    urlpatterns = patterns('')
```

Comment:
Currently it doesn't look like anything in the tests needs/uses these views, for any python/celery version combination.

If they're still useful for something besides running the test suite, we can leave them in. Or substitute equivalents if we want to still make django-celery optional (for celery >= 3.1) instead of a strict testing dependency.